nemo/collections/asr/models/classification_models.py

```diff
@@ -334,12 +343,18 @@ def _setup_feature_label_dataloader(self, config: DictConfig) -> torch.utils.dat
         )
 
     @torch.no_grad()
-    def transcribe(self, paths2audio_files: List[str], batch_size: int = 4, logprobs=False) -> List[str]:
+    def transcribe(
+        self,
+        audio: List[str],
+        batch_size: int = 4,
+        logprobs=False,
+        override_config: Optional[ClassificationTranscribeConfig] = None,
```

Comment:
doc string for `override_config` is missing

Reply:
I also think `override_config` is not a better name, how about just `config` or `infer_config`